api/pom.xml

```diff
@@ -17,183 +17,28 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <!-- This is just for now and will not work if the API has a separate release cycle than the rest. -->
         <groupId>org.eclipse.microprofile.config</groupId>
         <artifactId>microprofile-config-parent</artifactId>
         <version>2.1-SNAPSHOT</version>
     </parent>
 
-    <groupId>org.eclipse.microprofile.config</groupId>
     <artifactId>microprofile-config-api</artifactId>
     <name>MicroProfile Config API</name>
     <description>MicroProfile Config :: API</description>
-    <url>https://microprofile.io/project/eclipse/microprofile-config</url>
+
     <dependencies>
-        <dependency>
-            <groupId>biz.aQute.bnd</groupId>
-            <artifactId>biz.aQute.bnd.annotation</artifactId>
-        </dependency>
         <dependency>
             <groupId>jakarta.enterprise</groupId>
             <artifactId>jakarta.enterprise.cdi-api</artifactId>
-            <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>org.osgi</groupId>
-            <artifactId>org.osgi.service.cdi</artifactId>
+            <groupId>biz.aQute.bnd</groupId>
```

Comment:
Can you move this dependency to the parent pom as well, becuase all of the specs need this?

Reply:
Hum, I had a look into the other projects and I couldn't find any usages. Can you point me to where it is being used other than config?

Reply:
This was needed by OSGi runtime. They planned to integrate with all MP specs and consumed Config first. I have not seen more update then. Let's leave it for now and move the dependency to parent when they resume their activities.